eng/tools/RepoTasks/RepoTasks.csproj

```diff
@@ -15,14 +15,14 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="NuGet.Packaging" Version="5.9.0" />
+    <PackageReference Include="NuGet.Packaging" Version="$(NuGetVersioningPackageVersion)" />
     <PackageReference Include="Microsoft.Extensions.DependencyModel" Version="2.1.0" />
   </ItemGroup>
 
   <ItemGroup Condition="'$(TargetFramework)' == '$(DefaultNetCoreTargetFramework)'">
-    <PackageReference Include="Microsoft.Build.Framework" Version="16.9.0" />
-    <PackageReference Include="Microsoft.Build.Tasks.Core" Version="16.9.0" />
-    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="16.9.0" />
+    <PackageReference Include="Microsoft.Build.Framework" Version="$(MicrosoftBuildFrameworkPackageVersion)" />
+    <PackageReference Include="Microsoft.Build.Tasks.Core" Version="$(MicrosoftBuildFrameworkPackageVersion)" />
+    <PackageReference Include="Microsoft.Build.Utilities.Core" Version="$(MicrosoftBuildFrameworkPackageVersion)" />
```

Comment:
1. Use existing properties in this file.
2. Suggest adding `$(MicrosoftBuildTasksCoreVersion)` and `$(MicrosoftBuildUtilitiesCoreVersion)` in eng/Versions.props. We don't normally do these 1:many version mappings outside that file and eng/Dependencies.props.

Reply:
Will adjust per response to https://github.com/dotnet/aspnetcore/pull/32790#discussion_r642810011